src/App.js

```diff
@@ -52,7 +36,7 @@ export default function App() {
       <button onClick={() => setModal(true)}>add new task</button>
       {modalVisible && <Modal onClose={onClose} />}
       <div className="header row">
-        {days.map(day => (
+        {fullDays.map(day => (
```

Comment:
`fullDays` meaning an array of all the days from sunday to sat. Open to suggestions on the naming here